modules/core/src/main/java/org/apache/synapse/unittest/CommonUtils.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
```

Comment:
Formatting issue in license header
let's correct rest as well in this PR

Reply:
License header issue fixed in https://github.com/wso2/wso2-synapse/pull/1452